server/service/osquery_utils/queries.go

```diff
@@ -40,6 +40,10 @@ type DetailQuery struct {
 	// Platforms is a list of platforms to run the query on. If this value is
 	// empty, run on all platforms.
 	Platforms []string
+	// SoftwareOverrideMatch is a function that can be used to override a software
+	// result.  If the function returns true, the software will be overridden with
+	// the results of the Query.
```

Comment:
If the function returns `false`, it is not overriden but still appended to the results, right?

Reply:
That's correct, I clarified the comment here